src/modules/ffmpeg/producer/av_producer.cpp

```diff
@@ -223,7 +223,7 @@ struct Filter
                 filter_spec = "anull";
             }
 
-            filter_spec += (boost::format(",aresample=async=1000:first_pts=%d:min_comp=0.01,asetrate=r=%d,"
+            filter_spec += (boost::format(",aresample=async=1000:first_pts=%d:min_comp=0.01,aresample=%d,"
```

Comment:
Looks to me as this PR uses resample filter twice now.
Can this be combined into only one filter step?

Maybe
`aresample=osr=48000:async=1000:first_pts=%d:min_comp=0.01`
would be a better solution?

Reply:
i think no, first filter Stretch/squeeze the data to make it match the timestamps, second one changes sample rate to 48kHz.

Reply:
as i remember async is not a parameter, it's another filter which computes new sample rate to match sync, and its results are used to adjust aresample output sample rate.

Reply:
You can combine

Reply:
async is a parameter of the aresample-filter

Reply:
yes, you are right. Everyhing works. I will make new PR to fix it.